File: dienmayxanhapi/dienmayxanhapi/Model/group.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace dienmayxanhapi.Model
{
  public class group
  {
    public int? _id { get; set; }
    public string tengroup { get; set; }
    public IList<danhsachquyentruycap> danhsachquyentruycap { get; set; }
  }

  public class danhsachquyentruycap
  {
    public int? _id_quyen { get; set; }
  }
}

File: src/app/model/group.ts
export class grp {
    _id: number;
    tengroup: string;
    danhsachquyentruycap: Array<quyentruycap>;
    constructor(_id: number, tengroup: string, danhsachquyentruycap: Array<quyentruycap>) {
        this._id = _id;
        this.tengroup = tengroup;
        this.danhsachquyentruycap = danhsachquyentruycap;
    }
}

export class quyentruycap
{
    _id_quyen: number;

    constructor(_id_quyen: number) {
        this._id_quyen=_id_quyen;
    }
}